cognite/client/data_classes/_base.py

```diff
@@ -43,29 +48,34 @@ def dump(self, camel_case: bool = False) -> Dict[str, Any]:
         return dumped
 
     @classmethod
-    def _load(cls, api_response):
+    def _load(cls, api_response: Dict[str, Any]) -> "CogniteResponse":
```

Comment:
why are we not returning an error here instead of a string?

Reply:
I don't follow. What error is it we should be returning?

Reply:
this was before I asked you why are we putting the return value as strings. ignore.